storio-content-resolver-annotations-processor/src/main/java/com/pushtorefresh/storio/contentresolver/annotations/processor/generate/PutResolverGenerator.java

```diff
@@ -115,11 +115,20 @@ private MethodSpec createMapToContentValuesMethodSpec(@NotNull final StorIOConte
                 .addCode("\n");
 
         for (final StorIOContentResolverColumnMeta columnMeta : storIOContentResolverTypeMeta.columns.values()) {
+            final boolean primitive = columnMeta.element.asType().getKind().isPrimitive();
+            final boolean needIgnoreNull = columnMeta.storIOColumn.ignoreNull();
```

Comment:
does it make sense to throw or at least log if user annotated primitive type with @IgnoreNull?